modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectEntryModelListener.java

```diff
@@ -101,9 +101,44 @@ public void onAfterRemove(ObjectEntry objectEntry)
 			throw new ModelListenerException(portalException);
 		}
 
-		_executeObjectActions(
-			ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE, objectEntry,
-			objectEntry);
+		ObjectEntry objectEntry1 = objectEntry;
```

Comment:
You don't need to create this variable before the Try/Catch.
You can use the `objectEntry` variable whenever you want and create another variable `rootObjectEntry`.